sdk/storage_blobs/src/service/operations/find_blobs_by_tags.rs

```diff
@@ -40,13 +45,20 @@ impl FindBlobsByTagsBuilder {
     }
 }
 
+/// ` AND ` in the spec can be converted to `+AND+`, however, azure rest api won't accept it unless we do a `%20AND%20`
+fn make_url_compatible_with_api(url: &mut Url) {
+    let compatible_query = url.query().map(|q| q.replace("+", "%20"));
+
+    url.set_query(compatible_query.as_deref());
+}
```

Comment:
General note, requiring `%20` instead of `+` as the proper encoding for a space character is service-wide. Any URL encoding done by the library for storage requests needs to do this. Probably beyond the scope of this PR, but good future reference.